### PR TITLE
fix: added config param to axon and some minor changes

### PR DESCRIPTION
**Pull request by sergioreina**

### CODE REVIEW

File: neurons/miner.py [hunk 1]
```diff
@@ -299,10 +291,10 @@ async def ping(synapse: storage.protocol.Ping) -> storage.protocol.Ping:
     while True:
         try:
             # Below: Periodically update our knowledge of the network graph.
-            if step % 5 == 0:
+            if step % 300 == 0:
```
Comment (salahawk) [inline neurons/miner.py:294]: Why are you changing this to 300, @sergioreina ?

Comment (sergioreina) [reply neurons/miner.py:294]: Sorry about that, @salahawk. I just felt like having that information pop up every five seconds was a bit too annoying in the logs, so I switched it to every five minutes. If it's causing any issues or if you prefer it the way it was, feel free to disregard it.

Comment (salahawk) [reply neurons/miner.py:294]: > Sorry about that, @salahawk. I just felt like having that information pop up every five seconds was a bit too annoying in the logs, so I switched it to every five minutes. If it's causing any issues or if you prefer it the way it was, feel free to disregard it.

It takes dozens of minutes to benchmark - so it's going to be too long for one set-weight interval.

Comment (salahawk) [reply neurons/miner.py:294]: Can you please revert this change and commit again? 🙏

**Pull request merged**